gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/avatarpicker/AvatarPicker.kt

```diff
@@ -252,6 +255,20 @@ private fun AvatarPickerAction.handle(
         }
 
         AvatarPickerAction.InvokeAuthFailed -> onSessionExpired()
+
+        is AvatarPickerAction.AvatarDeletionFailed -> {
+            scope.launch {
+                if (snackState.showQESnackbar(
```

Comment:
❓ Our default behavior when the snackbar has action is to don't dismiss. I wonder if we should add a limited duration, maybe long enough for the user to tap "Try again" but not to keep that toast on the screen. Wdyt?

Reply:
Yeah, sounds good to me!

Reply:
👍 Applied `SnackbarDuration.Long`